static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -161,30 +145,35 @@ define([
                 && test.canRun() && !expired && isTestSwitchedOn(test));
     }
 
+    function getId(test) {
```

Comment:
I would go as far as creating a functional because that pattern can be seen across the whole codebase. So:

```
function getProp(prop) {
  return function (_) {
    return _[prop];
  }
}

var getId = getProp('id');
...
```